snippets/cdc-acm-console/snippet.yml

```diff
@@ -0,0 +1,4 @@
+name: cdc-acm-console
+append:
+  OVERLAY_CONFIG: cdc-acm-console.conf
+  DTC_OVERLAY_FILE: cdc-acm-console.overlay
```

Comment:
> I don't personally see the merit in excluding files, if you want a different set of files, create a new snippet. Likewise in a project with build files, if you want to exclude files, create a new project or create a separate configuration directory and set APP_CONFIGURATION_DIR to point to those files instead of hacking around cmake files trying to alter zephyr variables.